go-controller/pkg/ovn/master.go

```diff
@@ -155,6 +155,19 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 			return err
 		}
 	}
+
+	oc.SCTPLoadBalancerUUID, stderr, err = util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "load_balancer", "external_ids:k8s-cluster-lb-sctp=yes")
+	if err != nil {
+		klog.Errorf("Failed to get sctp load-balancer, stderr: %q, error: %v", stderr, err)
+		return err
```

Comment:
s/Failed/failed/

Reply:
that is a logging cmd not a return err. I thought capitalization only mattered for returning an error